airflow/www/api/experimental/endpoints.py

```diff
@@ -70,7 +70,8 @@ def add_deprecation_headers(response: Response):
     return response
 
 
-api_experimental.after_request(add_deprecation_headers)
+# This is really experimental. We do not care too much about typing here
+api_experimental.after_request(add_deprecation_headers)  # type: ignore[arg-type]
```

Comment:
Maybe we should start calling this _deprecated_ API. (It’d still show _experiemental_ in code for compatibility for now, but we can call it deprecated in docs and comments.)

Reply:
I tihnk we already do :). https://airflow.apache.org/docs/apache-airflow/stable/deprecated-rest-api-ref.html

Reply:
This comment still says experiemental though

Reply:
crossed my mind indeed. Fixed.